client/src/components/SaladListComponent.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react"
+import SaladTile from "./SaladTile"
+
+
+const SaladListComponent = (props) => {
+
+    const [salads, setSalads] = useState([])
+
+    const getSalads = async() =>{
+        try{
+            const response = await fetch("/api/v1/salads")
+            if(!response.ok){
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const responseBody = await response.json()
+            setSalads(responseBody.salads)
+        }catch(error){
+            console.error(`Error in Fetch: ${error.message}`)
+        }
+    }
+
+    useEffect(()=> {
+        getSalads()
+    }, [])
+
+    const saladItems = salads.map((salad)=> {
+        
+        return (
```

Comment:
can remove the empty line before the return
https://github.com/airbnb/javascript#whitespace--padded-blocks